imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -166,83 +166,34 @@ def read_in_flight_cal_data() -> pd.DataFrame:
     return empty_df
 
 
-def calculate_calibration_factor(acquisition_time: float) -> npt.NDArray:
+def interp_func(x: np.ndarray, xp: np.ndarray, fp: np.ndarray) -> npt.NDArray:
```

Comment:
This is a small helper and should probably be inlined into the one function that uses it or renamed from the generic `interp_func`